### PR TITLE
TextAdventure game without enc ftm

### PR DESCRIPTION
**Pull request by annejonas2**

### CODE REVIEW

**Comment by danielsgriffin** [comment]
Good.

check formatting, goal being something like this:
```
1	2	3	4	5	6	7	8	9	10
11	12	13	14	15	16	17	18	19	20
```
v.:
```
1 2 3 4 5 6 7 8 9 10 

12 13 14 15 16 17 18 19 20 
```

File: HW05_ex00_TextAdventure.py [hunk 1]
```diff
@@ -2,75 +2,72 @@
...
     print "The bear has a bunch of honey."
     print "The fat bear is in front of another door."
-    print "How are you going to move the bear?"
+    print "How is " + user + " going to move the bear?"
     bear_moved = False
 
     while True:
         next = raw_input("> ")
 
-        if next == "take honey":
```
Comment (Neeragrover) [inline HW05_ex00_TextAdventure.py]: This condition will always be true

File: HW05_ex00_TextAdventure.py [hunk 2]
```diff
@@ -2,75 +2,72 @@
...
-            dead("The bear looks at you then slaps your face off.")
-        elif next == "taunt bear" and not bear_moved:
-            print "The bear has moved from the door. You can go through it now."
+        if next == "take" or "honey":
+            dead("The bear looks at " + user + " then slaps their face off.")
```
Comment (Neeragrover) [inline HW05_ex00_TextAdventure.py]: This always evaluates to true

Comment (annejonas2) [inline HW05_ex00_TextAdventure.py]: thanks!

**Pull request closed without merging**